build/index.js

```diff
@@ -166,12 +166,13 @@ const npm = async (...args) => {
         { ...process.env, FORCE_COLOR: 1 }
     );
     const elapsed = new Date() - start;
-    logDetail(`NPM finished in ${elapsed} ms`);
+    const realCommand = args[args.length - 1];
+    logDetail(` ${ANSIFont2("npm " + realCommand)} finished in ${elapsed} ms`);
 };
 
 async function npmInstall(name) {
     logDetail(`${name}: Installing dependencies ...`);
-    await npm("ci");
+    await npm("--no-audit", "--prefer-offline", "ci");
```

Comment:
Correct. By itself, it does not have a lot of impact, it's only combined with the cache when it really hits, giving it a bit of an edge. In general, if you install modules auditing them is marginal, and does not really take a lot of time. But if modules are already installed (because they have been cached, for instance), this really makes a difference, because when they're instlaled auditing is essentially the only thing that's done.